src/standard-configuration/variables.ts

```diff
@@ -41,14 +41,14 @@ export const parseVariable = (
   const res =
     'template' in config && isDefined(config.template)
       ? { ...omit(config, 'template'), ...config.template(stateObj, hass) }
-      : <ListVariableConfig | LevelVariableConfig | TextVariableConfig>{ ...config };
+      : ({ ...config } as ListVariableConfig | LevelVariableConfig | TextVariableConfig);
```

Comment:
The changes to this file were done “manually” by me, in order to address the following `npm run lint` errors:
```
$ npm run lint
...
nielsfaber/scheduler-card/src/standard-configuration/variables.ts
  44:9   error  Use 'as ListVariableConfig | LevelVariableConfig | TextVariableConfig' instead of '<ListVariableConfig | LevelVariableConfig | TextVariableConfig>'  @typescript-eslint/consistent-type-assertions
  51:12  error  Use 'as TextVariableConfig' instead of '<TextVariableConfig>'                                                                                        @typescript-eslint/consistent-type-assertions
```